src/components/TodoItem/TodoItem.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable jsx-a11y/label-has-for */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class TodoItem extends Component {
+  handleClickCheckBox = ({ target }) => {
+    const id = +target.id[target.id.length - 1];
```

Comment:
add explicit type conversion or wrap the first action in parentheses

Reply:
change everything according to this comment

Reply:
> add explicit type conversion or wrap the first action in parentheses
> change everything according to this comment

refactored